src/App.jsx

```diff
@@ -0,0 +1,44 @@
+import {useState, useEffect} from 'react'
+import Header from './components/Header'
+import LocationSelector from "./components/LocationSelector/index.jsx";
+import ResultsDisplay from "./components/ResultsDisplay/index.jsx";
+import Footer from "./components/Footer/index.jsx";
+import {BrowserRouter} from "react-router-dom";
+
+function App() {
+
+    const [stations, setStations] = useState([])
+    const [selected, setSelected] = useState(null)
+    const [destinationSelected, setDestinationSelected] = useState(null)
+
+
+    const fetchStations = async () => {
+        const response = await fetch('http://localhost:3000/stations')
```

Comment:
It would be a good idea to take out the `http://localhost:3000` part of this and put it as a variable at the top of the file, to make it easier to change when it's deployed to plesk